docker/dev.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+COMPOSE_FILE=dev.yml
+COMPOSE_CMD="docker compose -f $COMPOSE_FILE"
+
+export REPO_PATH="../"
+export SECURITY_PLUGIN_REPO_PATH="../../wazuh-security-dashboards-plugin"
+export WZD_CONFIG_PATH="./config/wzd/opensearch_dashboards.yml"
+export NODE_VERSION=$(cat $REPO_PATH/.nvmrc)
+export OPENSEARCH_VERSION=$1 || $(node -e "console.log(require('$REPO_PATH/package.json').version)")
```

Comment:
**thought**: if NodeJS is required to run the script, then maybe we could mention this optional requirement in the readme.